test_scripts/Polices/Related_HMI_API/ATF_GetUserFriendlyMessage_with_Language.lua

```diff
@@ -0,0 +1,67 @@
+
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies]: SDL.GetUserFriendlyMessage ("language")
+-- [HMI API] SDL.GetUserFriendlyMessage request/response
+--
+-- Description:
+-- 1. Precondition: SDL is started, HMI is started, App is registered
+-- 2. Steps: Activate App, in SDL.GetUserFriendlyMessage parameter "language" should be present
+--
+-- Expected result:
+-- PoliciesManager must respond with the information from the sub-section with the name correspondent to "language" value
```

Comment:
Please write what is the expected result in terms of communication.
You can use verification criteria from requirement.